src/main/java/com/github/redouane59/twitter/ITwitterClient.java

```diff
@@ -217,6 +221,33 @@
      */
     List<ITweet> searchForTweetsArchive(String query, LocalDateTime fromDate, LocalDateTime toDate);
 
+    /**
+     * To be implemented calling https://api.twitter.com/2/tweets/search/stream
+     * @param consumer
```

Comment:
If the JavaDoc does not add anything, don't add it.

```suggestion
```

Reply:
I thought it will fail if i didn't have any description, thanks !